blog/.gitignore

```diff
@@ -1,2 +1,3 @@
 _site
 .sass-cache
+.jekyll-metadata
```

Comment:
The output from jekyll build is stored in `/public/blog` correct? Should that also be git ignored?